### ISSUE TITLE
Weather Meter skin is broken; weather.com API key has changed

### ISSUE THREAD

**Issue by SilverAzide** [SilverAzide/Gadgets#34]
All skins based on JSMorley's weather.com JSON templates are broken as of 1/10/2022.  TWC has changed their API to a different value.  To fix this, open the WeatherComJSONVariables.inc file and change the APIKey variable as follows:
```ini
APIKey=21d8a80b3d6b444998a80b3d6b1449d3
```
Save the file and refresh the skin, and all should work.  This will be corrected in the next release.

**Comment by SilverAzide** [comment]
Corrected in v7.2.0.

**Comment by Sobanim** [comment]
The problem reappeared yesterday. Is there a fix?

**Comment by miquelfire** [comment]
A release was made yesterday to fix it already. Three other issues were already made about it.

**Comment by SilverAzide** [comment]
> The problem reappeared yesterday. Is there a fix?

Of course.  [Get the latest here.](https://github.com/SilverAzide/Gadgets/releases/latest)
